test/functional/tests/conftest.py

```diff
@@ -118,6 +118,7 @@ def pytest_runtest_teardown():
                 if TestRun.executor.is_active():
                     TestRun.executor.wait_for_connection()
                 Udev.enable()
+                kill_all_io()
```

Comment:
Does this commit update framework as well or is this line the only chagne?

Reply:
After rebase artifact, modified the message to reflect what's actually happening